src/Nethermind/Nethermind.Runner/Ethereum/Steps/LoadChainspec.cs

```diff
@@ -40,12 +42,33 @@ public Task Execute()
         
         private void LoadChainSpec()
         {
+
             IInitConfig initConfig = _context.Config<IInitConfig>();
-            if (_context.Logger.IsInfo) _context.Logger.Info($"Loading chain spec from {initConfig.ChainSpecPath}");
+
+            if (!string.IsNullOrEmpty(initConfig.ChainSpecDirectory))
+            {
+                chainspecDir = initConfig.ChainSpecDirectory;
+
```

Comment:
storing chainSpecDirectory and then calling initConfig.ChainSpecDirectory over and over